src/qt/bitcoingui.cpp

```diff
@@ -973,7 +973,7 @@ void BitcoinGUI::createActions()
 	votingAction->setMenuRole(QAction::TextHeuristicRole);
 
 
-	votingReservedAction = new QAction(QIcon(":/icons/bitcoin"), tr("&Reserved - Voting Linux"), this);
+    votingReservedAction = new QAction(QIcon(":/icons/bitcoin"), tr("&Voting Linux"), this);
```

Comment:
Maybe we can change this to just "Voting"?
Also, rename the variable votingAction.